synapse/docker/__init__.py

```diff
@@ -1,9 +0,0 @@
-import os
-
-from synapse.cortex import openurl
-
-
-def getSyncCore():
```

Comment:
This was unused and really just not the right thing to do.

Reply:
A better way to achieve this is to map in a directory (which contains the dmon file/config file/etc that the user would like to run/use) as a docker volume.